README.md

```diff
@@ -515,6 +515,23 @@ In order to do that, there are two steps needed:
 - deriving the records that should be parcelable with the keyword parcelable: `deriving(parcelable)`
 - run Djinni with the following flag `--java-implement-android-os-parcelable true`
 
+## Support for `-fvisibility=hidden` and `-fvisibility=default`
+
+You can pass `-fvisibility=hidden` or the `-fvisibility=default` flags to your compiler, Djinni handles both cases well.
+
+The symbols that are belonging to the public interfaces that are generated by Djinni are automatically defined to be visible symbols.
```

Comment:
I feel like it's not obvious that making these symbols always public is the right behavior for all users.  E.g. if Djinni symbols are only used internally to a larger library, they wouldn't need to be exported.  In the use case we've used at Dropbox, we build all our C++ code into a single library on Android, so only the JNI functions need to be exported.